integration/resource_lacework_resource_group_test.go

```diff
@@ -0,0 +1,40 @@
+package integration
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/stretchr/testify/assert"
+)
+
+// TestResourceGroupCreate applies integration terraform:
+// => '../examples/resource_lacework_resource_group'
+//
+// It uses the go-sdk to verify the created resource group,
+// applies an update with new description and destroys it
+//
+// Note: This test will only work in environments where RGv2 FF is enabled
+func TestResourceGroupCreate(t *testing.T) {
```

Comment:
Is it possible to enable RGv2 on the CustomerDemo? Otherwise this may need disabled for CI to pass.

```suggestion
func _TestResourceGroupCreate(t *testing.T) {
```

Reply:
Yeah, I'm planning to ask PMs if this is ok to do or not before enabling it.

Reply:
I've enabled RGv2 FF in TECH-ALLY subaccount